share/completions/nvram.fish

```diff
@@ -0,0 +1,12 @@
+# completion for nvram (macOS)
+
+function __fish_nvram_variables
+    command nvram -p
```

Comment:
Well, not really necessary to define a function here -- you could run `nvram -p` directly.